jsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "compilerOptions": {
+        "checkJs": true,
+        "module": "es2020",
+        "target": "es2020",
+    }
+}
```

Comment:
can you format with prettier?

We can update the `lint` scripts 

https://github.com/octoherd/create-octoherd-script/blob/cf9eb5e96be7de6da7af433a150a0f624322b7a2/package.json#L18-L19

to update all `*.json` and `*.md` files in the root folder 

I'm reluctant about adding config files as you know, but I think it's worth it in this case. I wish we could add a comment to explain why it is here for people who don't know. Do you know if `jsconfig.json` supports json5 by default?

Reply:
> can you format with prettier?
> 
> We can update the `lint` scripts
> 
> https://github.com/octoherd/create-octoherd-script/blob/cf9eb5e96be7de6da7af433a150a0f624322b7a2/package.json#L18-L19
>
> to update all `*.json` and `*.md` files in the root folder

could you address that please?